plugins/DBStats/templates/index.twig

```diff
@@ -42,7 +42,7 @@
     <tbody>
     <tr>
         <td>
-            <h2>{{ 'DBStats_TrackerTables'|translate }}</h2>
+            <h2 class="secondary">{{ 'DBStats_TrackerTables'|translate }}</h2>
```

Comment:
It's also used somewhere else. Eg in sitesManager. We try to detect the alternative h2 automatically (the first H2 in a page has a border-bottom and less margin-top/bottom and all others have more margin-top and no border-bottom) but it currently only works for H2 that are within the same element (`:first-of-type`). We can maybe go with a JS solution and set this class automatically but wanted to keep it simple for now. If I find time today will have a look. It is not an h3